distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> For the distribution, it would therefore be enough if you simply have the feature openhab-runtime-base pre-installed - ok?

OK, I will update the repo config files and add the shk repo to the online, so that it becomes 'fully' online.